app/library/urls.py

```diff
@@ -4,4 +4,6 @@
 urlpatterns = [
     path('', include('realDealz.urls')),
     path('/admin/', admin.site.urls),
-]
\ No newline at end of file
+]
+
+#confused about tis page in why templates has it if realDealz has one already  
```

Comment:
Django project can have multiple apps within a single project read the documentation.